### ISSUE TITLE
How to deal with webview-bridge when running in a browser?

### ISSUE THREAD

**Issue by JonasDoe** [gronxb/webview-bridge#89]
In our case we have an ordinary (Next.js) web app and a mobile version of it, the latter utilizing webview-bridge. Right now, it seems like we have to check the environment before creating a link bridge and even have to "mock" properties like `BridgeProvider` and methods like `isNativeMethodAvailable` in the web app to avoid the warning `[WebViewBridge] Not in a WebView environment` or even `Method ... is not defined` errors.
It would be great if this would be handled by the webview-bridge instead, given the scenario (web app + mobile WebView app) is probably not really uncommon. Or am I missing a solution which is already there?

**Comment by gronxb** [comment]
It seems like what you are looking for is the initialBridge option.

Please check here:
https://gronxb.github.io/webview-bridge/non-react-native-environment.html

If this is not what you meant, please leave a comment.

**Comment by gronxb** [comment]
It seems fine for the warning to appear only when the debug option is set to true. 👍

What do you think?
I should be able to work on it by tomorrow.

**Comment by JonasDoe** [comment]
Okay, I took a look, but I'm not sure I could really speed things up b/c there are too many open questions for me:
- `debug` doesn't seem to exist as an option in `LinkBridgeOptions` right now. I can add it, but maybe I misunderstood your idea.
- In `LinkBridgeOptions`, we only log the warning `options.onFallback`. I don't grasp the concept behind this check, but ofc. I could just add check for `options.debug` and hope everything is alright with that.
- The warning is also logged in `packages/web/src/registerWebMethod.ts#registerWebMethod`, which doesn't even accept any options. I guess you might want to keep the warning here, though, since it's deprecated, but I'm not sure.
- I'm having some trouble with the tool chain. Imports from peer projects won't work until I add a `/src` to the import path. (I've never worked with pnpm - or corepack - before, so I might utilize it the wrong way). I could create a PR without this anyway ofc, but I couldn't run the linter.

Long story short: I'm afraid there might be too many questions here before I could do anything. :(

**Comment by gronxb** [comment]
Thank you for checking! I’ll try to get to it as soon as I have some time. 👀

**Comment by gronxb** [comment]
@JonasDoe 

Please update all versions to 1.7.5.

Additionally,
```ts
createLinkBridgeProvider({
  debug: false, // default false
});
```
or
```ts
linkBridge({
  debug: false, // default false
});
```

will not output the console logs as you expect. Since the default value is set to false, you don’t need to specify this field unless you want to. However, during development, make sure to set debug: true to check the logs.

**Comment by JonasDoe** [comment]
That fixed it! Thank you again for the fast response and the fast fix! :)
